script/functions.sh

```diff
@@ -13,7 +13,7 @@ MAINTENANCE_BRANCH=`cat maintenance-branch`
 
 # Don't allow rubygems to pollute what's loaded. Also, things boot faster
 # without the extra load time of rubygems. Only works on MRI Ruby 1.9+
-if is_mri; then
+if is_mri_192_plus; then
```

Comment:
In theory this is ok, but makes no sense for rspec-rails as its always 192 plus.